app/views/smart_answers/_previous_answer.html.erb

```diff
@@ -12,7 +12,14 @@
     <% end %>
   </td>
   <td class="govuk-table__cell">
-    <%= link_to @presenter.change_collapsed_question_link(number_questions_so_far + 1), class: "govuk-link" do %>
+    <%= link_to @presenter.change_collapsed_question_link(number_questions_so_far + 1),
+      class: "govuk-link",
+      data: {
+        "module": "track-click",
+        "track-category": "Smart Answer Change Link",
+        "track-action":  question.title,
```

Comment:
This is a bit different to [the other PR](https://github.com/alphagov/smart-answers/pull/4513/files#diff-85c63786058d6c48626a6c1cc46ba973R29), where the `action` really is an "action". Is this just something we have to do to get the data out? I'm wondering if we could do a combined string in the label, as a way to keep it consistent.

Reply:
Yup - make sense! I've made it a single string in the event label.